src/plugins/pdfPlayer/plugin.js

```diff
@@ -265,7 +265,7 @@ export class PdfPlayer {
 
     renderPage(canvas, number) {
         this.book.getPage(number).then(page => {
-            Events.trigger(this, 'timeupdate');
+            Events.trigger(this, 'pause');
```

Comment:
I tested the rewind to the first page and found that the event triggering is placed completely wrong.
`renderPage` is called when the page needs to be rendered. It isn't called for some number of previous pages (cached).
For now, I thing `Events.trigger` should be at the end of the `next` and `previous` functions.